src/main/java/PrintLine.java

```diff
@@ -0,0 +1,12 @@
+public class PrintLine {
```

Comment:
The naming of the class usually will not be an action. I think such a class can be simplified into a method within the main.